aptos-move/framework/aptos-framework/sources/configs/Stake.move

```diff
@@ -908,8 +1033,25 @@ module AptosFramework::Stake {
             remaining_lockup_time = stake_pool.locked_until_secs - current_time;
         };
 
+        let total_stake_before_rewards =
+            Coin::value(&stake_pool.active) + Coin::value(&stake_pool.pending_inactive);
```

Comment:
i'm now paranoid if the stake + reward overflows, but that's orthogonal to this PR

Reply:
Yeah I have a task to revisit the balance type for Coin. u64 only covers up to ~20**19. If the coin has 10 decimals, initial supply of 1B coins is already 10^19 which is going to be very close to the the limit already.